Ruddarr/Views/MoviesView.swift

```diff
@@ -24,6 +28,15 @@ struct MoviesView: View {
 
         NavigationStack {
             Group {
+                ContentUnavailableView(
```

Comment:
Why is this up here and not in line 66? Is that because of the commits to `main`?

Reply:
cause Im an idiot, is that a good explanation? 😀

Was testing this when I didn't have an instance, never removed it, will do now.

Reply:
*when I had an instance and didn't wanna reset everything